tools/profiling/memory/memory_diff.py

```diff
@@ -97,23 +103,28 @@ def _run():
 text = ''
 if old is None:
     print(cur)
-    for key, value in sorted(cur.items()):
+    for key, value in cur.items():
         text += '{}: {}\n'.format(key, value)
 else:
     print(cur, old)
-    diff_size = 0
+    call_diff_size = 0
+    channel_diff_size = 0
     for scenario in _SCENARIOS.keys():
-        for key, value in sorted(_INTERESTING.items()):
```

Comment:
Want to confirm that it's fine to remove sorted? when it's sorted, the results are in a confusing order with channel and call results being mixed between each other.

Reply:
so the slightly scary thing is that python doesn't guarantee an ordering on these... maybe we can call them `call/client`, `call/server`, `channel/client`, `channel/server` or the like instead of the current `client call`

Reply:
Oh, was not aware of that, thanks. Changed how we call them to call/client